src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.datasource.platform=postgres
+# url format:  jdbc:postgressql://servername:port/dbname
+spring.datasource.url=${SPRING_DATASOURCE_URL}
+spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
+spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
+# Gets rid of an unimportant hibernate error message
+# https://github.com/pgjdbc/pgjdbc/issues/1102
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+# Recreate the database on each run
+# https://stackoverflow.com/questions/42135114/how-does-exactly-spring-jpa-hibernate-ddl-auto-property-works-in-spring/42147995
+# create, update
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
This could have been an env variable defaulting to 'update' so it would have worked better with heroku.

${HIBERNATE_CREATE_OPTION:update}